lib/dest/options.js

```diff
@@ -29,6 +33,18 @@ var config = {
     type: ['string', 'boolean'],
     default: false,
   },
+  // Symlink options
+  useJunctions: {
+    type: 'boolean',
+    default: function(file) {
+      var isDirectory = file.isDirectory();
+      return (isWindows && isDirectory);
```

Comment:
Actually, isn't it as simple as changing the default? Seems to me that the directory case will be forced to symype `'file'` anyway, and the call to `file.isDirectory()` will have to be done in `write-contents/write-symbolic-link` in any event, so no need to do it twice?

Sorry if I'm missing something obvious, it's late here and been one of those days :-)